Packs/GraphQL/Integrations/GraphQL/GraphQL.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict
+
+import urllib3
+from gql import Client, gql
+from gql.transport.requests import RequestsHTTPTransport
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+def execute_query(client: Client, args: Dict) -> CommandResults:
+    query = gql(args.get('query'))
+    result = client.execute(query)
+    command_results_args = {
+        'outputs': result,
+        'outputs_prefix': 'GraphQL'
+    }
+    if args.get('outputs_key_field'):
+        command_results_args['outputs_key_field'] = args.get('outputs_key_field')
+    return CommandResults(**command_results_args)
+
+
+def main() -> None:
+    command = demisto.command()
+    try:
+        params = demisto.params()
+        request_params = {
+            'url': params.get('url'),
+            'verify': not params.get('insecure', False),
+            'retries': 3,
+        }
+        if params.get('credentials'):
```

Comment:
Let's also add support for specifying a header as we did in other integrations so a user can authenticate with a token. Such as in the github api: https://developer.github.com/v4/guides/forming-calls/#authenticating-with-graphql

Reply:
@glicht for that i added the `headers` parameter, dont you think its enough?